### ISSUE TITLE
Scripting: keyboard keys

### ISSUE THREAD

**Issue by lolslayer** [Kromster80/kam_remake#366]
For scripting, when I want to add something in where you need to select certain functions, I Always use beacons and specific points on the map where you need to put the beacons to activate the functions. This of course isn't really user friendly.

It might be nice to have some keyboard keys reserved for scripting, that we got a function like: States.ScriptKey1(Player) so you can see what reserved key is pressed by what player. Even 4 keys can be very effective and extremely useful.

**Comment by Kromster80** [comment]
This would need to go through Key Mapping. Something alike "Script Key #1" "Script Key #2", etc. Cos we allow players to remap keys and we don't want these to be overwritten

**Comment by lolslayer** [comment]
Yeah, that was actually what I meant :)

**Comment by reyandme** [comment]
Good idea, will add this hotkeys then. By default no hotkeys for that, but player could map these functions to any key
